Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -547,7 +556,7 @@ uint64_t Connection::sendWithAsyncReply(T&& message, C&& completionHandler, uint
             T::callReply(*decoder, WTFMove(completionHandler));
         else
             T::cancelReply(WTFMove(completionHandler));
-    }, CompletionHandlerCallThread::MainThread));
+    }));
```

Comment:
This might be a behavioural difference in some esoteric current call sites, so care should be taken to change this. E.g add an local assert that all callers are in main thread, run layout tests, then make a self-contained change removing this parameter explaining that this was indeed a redundant parameter here.